Kitodo/src/main/java/org/kitodo/production/converter/IllustratedSelectItemConverter.java

```diff
@@ -30,7 +30,7 @@ public class IllustratedSelectItemConverter implements Converter, Serializable {
 
     @Override
     public Object getAsObject(FacesContext context, UIComponent component, String value) {
-        if (StringUtils.isNotEmpty(value)) {
+        if (StringUtils.isNotBlank(value)) {
```

Comment:
Is this and even on other places the replacement from `isNotEmpty` to `isNotBlank` indented? `isNotEmpty` is much faster and did not allow white spaces,  `isNotBlank` is even checking for white space usage and should only used if white spaces should be allowed.

There are more places where this replacement is happening but I'm writing this only once.

Reply:
In all places where a string is checked for emptiness, something is done with it afterwards if it is not empty. For example, an `enum` constant is looked up from it. I have not found a case where an empty string should be processed differently than a string that only contains non-printing characters. Therefore, I think `isNotBlank()` is the right function here. Correct me if I am wrong.

Regarding the runtime: If the string is `null` or empty, it is identical, if it consists of non-printing characters, it is theoretically O(𝑛), but if the first printing character is found, the test stops anyway, so I assume a runtime of O(1). In any case, this is nothing that should noticeably slow down the application!

Reply:
In case that the value is  `null` or an empty string the behaviour is identical. If the first character is a non white space there is one iteration in the `isBlank` method which can be ignored. The case that in the beginning of a string a lot of white spaces occur is not so reasonable/realistic in most usage scenarios.

So ignore my comment regarding the `isBlank` / `isEmpty` methods. I must read the wrong method in the past how the `isBlank` method is working.